Runtimes/Core/cmake/caches/Vendors/Apple/Darwin.toolchain.cmake

```diff
@@ -0,0 +1,116 @@
+# Toolchain configuration when compiling for Darwin Platforms
+set(CMAKE_SYSTEM_NAME Darwin)
+set(CMAKE_Swift_COMPILER_WORKS YES)
+set(CMAKE_C_COMPILER_WORKS YES)
+set(CMAKE_CXX_COMPILER_WORKS YES)
+
+find_program(XCRUN_EXECUTABLE NAMES "xcrun" REQUIRED)
+
+if(NOT CMAKE_OSX_SYSROOT)
+  if(DEFINED ENV{SDKROOT})
+    set(sdk_name $ENV{SDKROOT})
```

Comment:
Why is this `sdk_name`? The SDKROOT is the full path to the SDK right?

Reply:
not necessarily. `macosx` and `iphoneos` are valid values for the `SDKROOT`. The `xcrun` call below does the translation to the actual full path to the SDK.